network-api/networkapi/buyersguide/utils.py

```diff
@@ -41,3 +52,38 @@ def database_backwards(self, app_label, schema_editor, from_state, to_state):
 
     def describe(self):
         return "Update %s bases to %s" % (self.name, self.bases)
+
+
+def convert_pni_file_to_product_page_image(pni_product: Product, new_product_page: ProductPage):
+    """
+    Take an existing product and convert it's FileField image to a WagtailImage object.
+
+    pni_product must be an instance of a Product
+    new_product_page must be an instance of a ProductPage
+    """
+    # 1. Get the mimetype of the image.
+    mime = MimeTypes()
+    mime_type = mime.guess_type(pni_product.image.file.name)  # -> ('image/jpeg', None)
+    mime_type = mime_type[0].split('/')[1].upper()
```

Comment:
warning: this assumes `mime_type[0]` is splittable on `/`, even though `guess_type()` may return `None`

Reply:
Oh good catch. Ill add in JPEG as a default mime type if one isnt found. I'm not convinced a default JPEG mimetype is the right move here. I'm open to new ideas.